packages/prettier-plugin-java/README.md

```diff
@@ -0,0 +1,111 @@
+[![npm](https://img.shields.io/npm/v/prettier-plugin-java.svg)](https://www.npmjs.com/package/prettier-plugin-java)
+
+# prettier-plugin-java
+
+![Prettier Banner](https://raw.githubusercontent.com/prettier/prettier-logo/master/images/prettier-banner-light.png)
+
+Prettier is an opinionated code formatter which forces a certain coding style. It makes the code consistent through an entire project.
+
+This plugin allows the support of Java on Prettier.
+
+The plugin implementation is pretty straightforward as it uses [java-parser](../java-parser) (thanks to Chevrotain) visitor to traverse the **C**oncrete **S**yntax **T**ree and apply the format processing on each node (it uses Prettier API).
+
+## Installation
+
+### Pre-requirements
+
+Since the plugin is meant to be used with Prettier, you need to install it:
+
+`npm install prettier --save-dev`
+
+or
+
+`yarn add prettier --dev`
+
+### Install plugin
+
+`npm install prettier-plugin-java --save-dev`
+
+or
+
+`yarn add prettier-plugin-java --dev`
+
+### CLI
+
+If you installed Prettier globally and want to format java code via the CLI, run the following command:
+
+`npm install -g prettier-plugin-java`
+
+The plugin with be automatically loaded, check [here](https://prettier.io/docs/en/plugins.html#using-plugins) for more.
+
+## Usage
```

Comment:
I think the main usage would be via CLI not via programmatic API.

Reply:
I will just invert those two then.